lspsaga.norg

```diff
@@ -0,0 +1,93 @@
+@document.meta
```

Comment:
Can we remove this file?

Reply:
Yep! I used this to sort of get an overview of all things I need to do, but then accidentally committed it.